parity/configuration.rs

```diff
@@ -475,6 +475,11 @@ impl Configuration {
 		let signer_path = Configuration::replace_home(&self.args.flag_signer_path);
 		::std::fs::create_dir_all(&signer_path).unwrap_or_else(|e| die_with_io_error("main", e));
 
+		if self.args.flag_geth {
+			let geth_path = path::ethereum::default();
+			::std::fs::create_dir_all(geth_path.as_path()).unwrap_or_else(
+				|e| die!("error {}: {}", &geth_path.to_str().unwrap(), e));
```

Comment:
Error should be more descriptive. Non-expect users might see this and just reading "error: ..." isn't helpful.

Try "Error while attempting to create the .ethereum directory for geth mode: {}".